learn/what_is_meilisearch/features.md

```diff
@@ -44,6 +44,12 @@ If you make a search without inputting any query words, MeiliSearch will return
 
 Placeholder search is not affected by MeiliSearch's [default ranking rules](/reference/features/settings.md#ranking-rules)—only custom rules added by a user. If no custom rules have been set, the results are displayed in the order of their internal database position.
 
+## Phrase search
```

Comment:
@gmourier: I have preferred `phrase search` over `phrase query`, is that ok?

Reply:
I have used both until then! I have no particular input, the two seem good to me! I will update the feature name on the public roadmap. Thanks!